pvlib/iotools/pvgis.py

```diff
@@ -373,7 +373,7 @@ def read_pvgis_hourly(filename, pvgis_format=None, map_variables=True):
         return _parse_pvgis_hourly_json(src, map_variables=map_variables)
 
     # CSV: use _parse_pvgis_hourly_csv()
-    if outputformat == 'csv':
+    elif outputformat == 'csv':
```

Comment:
unnecessary, this code can't be reached if `outputformat` is `'json'`

Reply:
Changing it to ``elif`` just makes it read a bit nicer, but doesn't change anything functionally.